### PR TITLE
Adding temporary debugging messages to logs

### PR DESCRIPTION
**Pull request by luke-powers**
These will help determine if the random hopper dump is software based
or hardware based. Once this determination is made this commit can be
reverted.

### CODE REVIEW

File: miso_peripherals/nodes/numato_gpio_node [hunk 1]
```diff
@@ -41,7 +42,7 @@ class NumatoGPIONode(object):
         """
         with self.__lock:
             self.controller.falling_edge(req.gpio, req.duration)
-            return ToggleGPIOResponse(error=GogoRoboError(success=True))
+            return ToggleGPIOResponse(error=GogoRoobError(success=True))
```
Comment (wguffey) [inline miso_peripherals/nodes/numato_gpio_node:45]: typo

File: miso_peripherals/src/miso_peripherals/numato_gpio.py [hunk 2]
```diff
@@ -3,6 +3,8 @@
 from time import sleep
 from warnings import warn
 
+import rospy
```
Comment (wguffey) [inline miso_peripherals/src/miso_peripherals/numato_gpio.py:6]: looks like this rospy import never gets used

File: miso_peripherals/nodes/numato_gpio_node (hunk 1 above)
Comment (luke-powers) [reply miso_peripherals/nodes/numato_gpio_node]: Oof, thanks. Fixed.

File: miso_peripherals/src/miso_peripherals/numato_gpio.py (hunk 2 above)
Comment (luke-powers) [reply miso_peripherals/src/miso_peripherals/numato_gpio.py:6]: Fixed

**Review by wguffey — APPROVED** [review]

**Review by pramati — APPROVED** [review]

**Review by zzv2 — APPROVED** [review]

**Review by zzv2 — COMMENTED** [review]
Can you change this from logwarn to loginfo?

**Comment by luke-powers** [comment]
> Can you change this from logwarn to loginfo?
Updated.

**Comment by luke-powers** [comment]
Cleaned up fixups

**Review by zzv2 — APPROVED** [review]

**Pull request merged**